sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/command.py

```diff
@@ -34,11 +34,11 @@ def __init__(self, **kwargs):
         self._init = False
 
     @property
-    def compute(self) -> str:
+    def compute(self) -> Optional[str]:
```

Comment:
FWIW, it looks like the rtype for `environment` and `environment_variables` is also wrong, as they could get set to `None` in the ctor :)

Reply:
Good callout - I'll fix.